app/models/manageiq/providers/kubevirt/infra_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,23 @@
+class ManageIQ::Providers::Kubevirt::InfraManager::EventCatcher::Runner < ManageIQ::Providers::Kubernetes::ContainerManager::EventCatcher::Runner
+  ENABLED_EVENTS = {
+    'VirtualMachine'         => %w(Created Started Migrated SuccessfulCreate SuccessfulDelete ShuttingDown Killing),
+    'VirtualMachineInstance' => %w(Created Started Migrated SuccessfulCreate SuccessfulDelete ShuttingDown Killing)
+  }
```

Comment:
Not for this PR but as a follow-up we'll want to create event handler objects in manageiq-content for these so that customers can hook into these events for policy

Example: https://github.com/ManageIQ/manageiq-content/tree/master/content/automate/ManageIQ/System/Event/EmsEvent/Kubernetes.class

Reply:
https://github.com/ManageIQ/manageiq-content/pull/755